src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/tests/Block.test.js

```diff
@@ -133,7 +138,12 @@ test('Changing the type should call the onTypeChange callback', () => {
         </Block>
     );
 
-    block.find('SingleSelect').simulate('change', 'type2');
+    const selectButton = screen.queryByLabelText('button');
```

Comment:
we should go here with `getByText` or `getByRole` as there are multiple buttons we can go with `getAllByRole` or first select the correct parent. we should not add a `aria-label="button"` as that would hurt accessibility.